src/main/java/lotto/controller/LottoStore.java

```diff
@@ -0,0 +1,54 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.Map;
+
+public class LottoStore {
+
+    public static final int LOTTO_PRICE = 1000;
+
+    public void process() {
+        Lottos purchasedLottos = buyLotto();
+        WinningLotto winningLotto = decideWinningLotto();
+        Map<LottoRank, Integer> lottoResultStatistics = purchasedLottos.getStatistics(winningLotto);
+        printLottoResult(lottoResultStatistics, purchasedLottos);
+    }
+
+    public Lottos buyLotto() {
+        Money money = new Money(InputView.inputMoney());
+        Lottos purchasedLottos = new Lottos(calculateAffordableLottoTickets(money));
+        OutputView.printPurchasedLottos(purchasedLottos);
+        return purchasedLottos;
+    }
+
+    private WinningLotto decideWinningLotto() {
+        Lotto winningLotto = Lotto.from(InputView.inputWinningNumbers());
+        BonusBall bonusBall = new BonusBall(InputView.inputBonusNumber(), winningLotto);
+        return new WinningLotto(winningLotto, bonusBall);
+    }
+
+    public void printLottoResult(Map<LottoRank, Integer> lottoResultStatistics, Lottos lottos) {
+        double profitRate = calculateProfitRate(lottoResultStatistics, lottos.getSize());
+        OutputView.printLottoStatistics(lottoResultStatistics, profitRate);
+    }
+
+    public double calculateProfitRate(Map<LottoRank, Integer> lottoResultStatistics, int purchasedLottoCount) {
+        double initialCapital = purchasedLottoCount * LOTTO_PRICE;
+        double sum = 0;
+
+        for (LottoRank rank : lottoResultStatistics.keySet()) {
```

Comment:
```suggestion
        lottoResultStatistics.entrySet().stream()
                .mapToInt(entry -> entry.getKey().getPrizeMoney() + entry.getValue())
                .sum();
```
대략 이렇게 수정해볼수도있어요
keySet을 사용하셨는데, 결과적으로 value도 가져와야하는작업입니다.
그럼 keySet보다는 entrySet을 사용해보는건 어떨까요? key와 value에 다 접근가능합니다
(추가적으로 합계도 저런식으로 가능할것으로보입니다.)
<img width="696" alt="스크린샷 2021-02-19 오후 8 59 43" src="https://user-images.githubusercontent.com/20608121/108501970-657a4d00-72f5-11eb-988f-e5321d437342.png">

Reply:
이 부분은 처음에 entrySet으로 진행하였다가 체이닝이 너무 지저분한 것 같아서 저런형식으로 했습니다! 그래도 어차피 value를 가져와야 하는 작업이니깐 entrySet으로 하는게 다시보니 나은 것 같네요 추가로 Stream으로 처리도 해보겠습니다!